### PR TITLE
Add csharp code view

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: src/Blockcore.Explorer/ClientApp/src/app/explorer/contract-address/contract-address.component.html [hunk 1]
```diff
@@ -56,6 +56,10 @@ <h3><i class="fas fa-receipt"></i>&nbsp;&nbsp;Smart Contract Details</h3>
          <span>View Smart Contract</span>
          <span><a [routerLink]="['../../','contract-transaction', transaction.transactionId]">{{transaction.transactionId | slice:0:20}}</a></span>
       </div>
+      <div>
+         <span>Smart Contract Code</span>
+         <span><a [routerLink]="['../../','contract-code', transaction.contractAddress]">View C# Code</a></span>
```
Comment (sondreb) [inline src/Blockcore.Explorer/ClientApp/src/app/explorer/contract-address/contract-address.component.html]: View Source Code?

File: src/Blockcore.Explorer/ClientApp/src/app/explorer/contract-code/contract-code.component.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   </div>
+</div>
+
+<div *ngIf="transaction" class="box left">
+   <h3>C# Code</h3>
+
+   <!--<ngx-prism class="p-0" [language]="'csharp'" [code]="{{transaction.csharp}}">
+   </ngx-prism>-->
+
+   <span><pre>{{transaction.csharp}}</pre></span>
```
Comment (sondreb) [inline src/Blockcore.Explorer/ClientApp/src/app/explorer/contract-code/contract-code.component.html]: "sourceCode"

File: src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html [hunk 3]
```diff
@@ -64,11 +64,15 @@ <h3>Network Statistics</h3>
             <span>Difficulty</span>
             <span>{{blockchain.difficulty}}</span>
          </div>
+         <div *ngIf="blockchain.networkWeight > 0">
```
Comment (sondreb) [inline src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html:67]: not sure if "blockchain" is just a basic model, and I know I fail on this myself all the time, but this really should be a get method that says: "hasNetworkWeight": boolean instead.

**Review by sondreb — APPROVED** [review]
Great work again, looking forward to seeing this in action 👍🏼

File: src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html (hunk 3 above)
Comment (dangershony) [reply src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html:67]: A method where on the `ContractCodeComponent` class?

Comment (dangershony) [reply src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html:67]: lets fix this later

**Pull request merged**